README.md

```diff
@@ -1 +1,35 @@
-# sdcore-upf-snap
\ No newline at end of file
+# SD-Core User Plane Function (UPF) Snap
+
+A snap for the SD-Core User Plane Function
+
+## Usage
+
+### Install
+
+```bash
+sudo snap install sdcore-upf
+```
+
+### Configure
+
+Configure the upf bessd by editing the `/var/snap/sdcore-upf/common/upf.json` file.
+
+### Start
+
+Start the services:
+
+```bash
+sudo snap start sdcore-upf.bessd
+sudo snap start sdcore-upf.routectl
```

Comment:
Not sure if this has some practical implications, but normally, looking at the processes, `routectl` starts before `bessd`.

Reply:
Updated